engine/Assets/Scripts/Drivers/RotationalDriver.cs

```diff
@@ -281,6 +293,8 @@ public enum RotationalControlMode {
                         }
 
                         _fakedTheta = Mathf.Clamp(_fakedTheta, -180, 179);
+                        if (_fakedTheta == 179 || _fakedTheta == -180)
```

Comment:
Depending on how the limits are setup, the limit can be anything in between -180 and 179. Check for no change in _fakedTheta instead.